content/content-bundles/resources/types.properties

```diff
@@ -195,7 +195,7 @@ Copyright: It is your personal responsibility to verify that you have permission
 to upload the file(s) to this website. Text, graphics and other media files may all be subject to copyright control \
 even if your site is restricted to site members.
 instr.url			 = Copy-and-paste or type in the web address (URL) and then click 'Continue' at the bottom.
-instr.urls		 	= Add as many web links (URLs) as you like. If you change your mind about needing one of your web links, click the 'X' icon beside it. Press the 'Add Web Links Now' button when you have finished.
+instr.urls		 	= Add both the address and name for a web link (URL). Click the 'Add Web Link Now' button when you have finished.
```

Comment:
I can make the change to use sakai_create_url.vm instead of the urls vm but they are not identical (apart from the instr.url), presumably I can't assume people just want to move to the other template?